sstables/kl/writer.cc

```diff
@@ -663,7 +663,7 @@ void sstable_writer_k_l::consume_end_of_stream()
         features.disable(sstable_feature::NonCompoundRangeTombstones);
     }
     run_identifier identifier{_run_identifier};
-    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier));
+    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier), false);
```

Comment:
Better to use a bool_class, or even better a struct with a bool member. That naked `false` is less readable.

Reply:
Using `enum class` now.